<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_ServerConnectionDialog</class>
 <widget class="QDialog" name="w_ServerConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/appIcon.png</normaloff>:/icons/appIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="l_tittle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Serwer - konfiguracja</string>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="Line" name="line_top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>400</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_bottom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>400</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="l_address">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adres</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_port">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_address">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_port">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>105</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="b_refresh">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Przywróc poprzednie dane</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/reset.png</normaloff>:/icons/reset.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="b_test">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Testuj połączenie</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/test_connection.png</normaloff>:/icons/test_connection.png</iconset>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>380</width>
     <height>50</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
